modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -0,0 +1,111 @@
+[id="cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_{context}"]
+= Configuration of JSON log data for default Elasticsearch
+
+With the default Elasticsearch log store, you must create an index for each format if the JSON log entries _have different formats_.
+
+[IMPORTANT]
+====
+You must create a separate index for each different JSON log format. Otherwise, forwarding different formats to the same index can cause type conflicts and cardinality problems.
+====
+
+To provide a different index for each format, you configure the `ClusterLogForwarder` custom resource. You use a structure type to construct the index name from a category, Kubernetes label, or input selector.
```

Comment:
"...category, Kubernetes label, or input selector."  Kubernetes label is the only one of these three that I can find in the rest of this topic.  Should this list be updated to better reflect the following content? Or should these keywords be added to the following content?

Reply:
Updated. Thanks.